sound/soc/intel/boards/sdw_rt711_rt1308_rt715.c

```diff
@@ -614,7 +614,7 @@ static int get_sdw_dai_link_num(struct snd_soc_acpi_mach_params *mach_params)
 static void init_dai_link(struct snd_soc_dai_link *dai_links,
 			  int id, char *name, int playback, int capture,
 			  struct snd_soc_dai_link_component *cpus,
-			  char *cpu_name,
```

Comment:
@RanderWang sorry I am a little late to the party her for this machine driver. But this patch isnt the most straight-forward to review so I just had to go through the whole file to understand the change. So please ignore if some of these have been disucssed already

Reply:
in the ini_dai_link() function, why dont we just pass the pointer to the dai_link that is being initialized instead of dai_links and the id of the dai link in dai_links? That way, you''ll have one id in the args and its easy to understand

Reply:
Good idea ! In my first change, id was also used by other feature. But now it is not used, so we can remove id. Thanks! I will refine it.